flake.nix

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Ehkä tää on nixistille selkee näinkin, mut oisko hyvä ajatus laittaa readmehen jotain ohjetta et btw, voit asentaa ympäristön yms. nixillä näin: lorem ipsum dolor..?

Reply:
done